internal/generator/vector/output/kafka/passphrase.go

```diff
@@ -0,0 +1,18 @@
+package kafka
+
+import (
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/output/security"
+)
+
+type Passphrase security.Passphrase
+
+func (p Passphrase) Name() string {
+	return "passphraseTemplate"
+}
+
+func (p Passphrase) Template() string {
```

Comment:
This is fluentd syntax?

Reply:
not exactly, passphrase is from kafka sasl config

Reply:
I meant it requires `=` sign

Reply:
thanks, i updated the PR. it has an `=` sighn now